subprojects/packagefiles/libccp4c/meson.build

```diff
@@ -0,0 +1,104 @@
+# Meson file for libccp4c (CCP4 core libraries, C part only)
+project('libccp4c', ['c'],
+        version: '6.5.1',
+        meson_version: '>=0.49.0',
+        license: 'LGPL3')
+
+
+cc = meson.get_compiler('c')
+mdep = cc.find_library('m', required: true)
+
+
+# Note that the source code assumes that datadir='share'.
+# If it is not, programs using the library will not be able
+# to find 'environ.def'.
```

Comment:
You don't seems to install `environ.def`.

Reply:
I've hopefully addressed this with my comment on the review by @eli-schwartz - see above.